features/admin.applications.v1/pages/application-edit.tsx

```diff
@@ -87,12 +88,14 @@ const ApplicationEditPage: FunctionComponent<ApplicationEditPageInterface> = (
 
     const appDescElement: React.MutableRefObject<HTMLDivElement> = useRef<HTMLDivElement>(null);
 
-    const allowedScopes: string = useSelector((state: AppState) => state?.auth?.allowedScopes);
     const applicationTemplates: ApplicationTemplateListItemInterface[] = useSelector(
         (state: AppState) => state.application.templates);
     const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state.config.ui.features);
     const tenantDomain: string = useSelector((state: AppState) => state.auth.tenantDomain);
 
+    // Check if the user has the required scopes to update the application.
+    const hasUsersUpdatePermissions: boolean = useRequiredScopes(featureConfig?.applications?.scopes?.update);
```

Comment:
Convention has been `has<feature><action>Permissions`. Let's stick to that.

```suggestion
    const hasApplicationUpdatePermissions: boolean = useRequiredScopes(featureConfig?.applications?.scopes?.update);
```

Reply:
Fixed with e404ed9145c6589528141f048ce9eb70cc53b864